src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -81,15 +87,42 @@ public Tenant save(UUID consortiumId, Tenant tenantDto) {
     }
 
     checkTenantNotExistsAndConsortiumExistsOrThrow(consortiumId, tenantDto.getId());
-    Tenant savedTenant = saveTenant(consortiumId, tenantDto);
+    TenantEntity savedTenantEntity = saveTenantEntity(consortiumId, tenantDto);
+    var savedTenant = converter.convert(savedTenantEntity, Tenant.class);
 
     runInFolioContext(createFolioExecutionContextForTenant(tenantDto.getId(), currentTenantContext, folioMetadata),
-      () -> configurationClient.saveConfiguration(createConsortiaConfigurationBody(centralTenantId)));
```

Comment:
If there will be some issues with creating primary affiliation - we need allow re-run operation, lets add some force flag as request param that will allow creating primary affiliation operation if tenant already created previously.

Reply:
![image](https://github.com/folio-org/mod-consortia/assets/113523904/5cd1c232-20e5-4f77-ba62-3a01b8c398f4)
It can be fixed by add @SneakyThrows keyword inconsistent in db when error occurred

Reply:
it is still not working